bin/sub.js

```diff
@@ -40,7 +42,11 @@ exports.main = (exports.main = async() => {
   } else if (isdir === 'rf') {
     func.renamefile();
   } else {
-    // Invalid response in isdir
-    return 2;
+    console.log(
+      chalk.green(
+        'Commands for-\n 1. Directory\n Create: d\n Delete: dld\n' +
```

Comment:
Don't write it under `else` section. It should be inside another `else if` .

Reply:
use some other command like else if ( isdir === 'h' || isdir === 'help'). Something like this